extensions/README.md

```diff
@@ -19,6 +19,7 @@ _Draft Khronos extensions are not ratified yet._
 * [MSFT_texture_dds](2.0/Vendor/MSFT_texture_dds/README.md)
 * [MSFT_packing_normalRoughnessMetallic](2.0/Vendor/MSFT_packing_normalRoughnessMetallic/README.md)
 * [MSFT_packing_occlusionRoughnessMetallic](2.0/Vendor/MSFT_packing_occlusionRoughnessMetallic/README.md)
+* [DRACO_animation_compression](2.0/Vendor/DRACO_animation_compression/README.md)
```

Comment:
Should this use the `GOOGLE` prefix?  Or do you want to use a `DRACO` prefix?  We have been allowing prefixes for both companies and projects.

Reply:
I'm fine with the DRACO prefix. But if people would rather us change to GOOGLE, that would be fine with me as well.

Reply:
Given `KHR_draco_mesh_compression`, would `DRACO_animation_compression` ultimately then be `KHR_draco_animation_compression`? If so, it makes more sense to go `GOOGLE_draco_animation_extension` now, IMO...